dns/UnboundBL/src/opnsense/mvc/app/models/OPNsense/UnboundBL/UnboundBL.xml

```diff
@@ -0,0 +1,25 @@
+<model>
+    <mount>//OPNsense/UnboundBL</mount>
+    <version>0.2</version>
+    <description>DNSBL for Unbound DNS</description>
+    <items>
+        <general>
+            <enabled type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </enabled>
+                <blacklist type="CSVListField">
+                    <Required>Y</Required>
```

Comment:
Required Y and no default value will lead to trouble, I'm getting an error here.